installer/ansible/clean.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 The OpenSDS Authors.
```

Comment:
please update with "Copyright 2022 The SODA Authors." in all files.

Reply:
done